### PR TITLE
Revise AUC mapping

### PR DESCRIPTION
**Pull request by jacobthill**
## Why was this change made?

I encountered an [error](https://github.com/sul-dlss/dlme-transform/issues/416) in the process of mapping that needs to be resolved before I can continue

### CODE REVIEW

**Comment by aaron-collier** [comment]
This is an interesting situation and possibly a bug in traject. We may want to schedule some pairing time around a possibly improved approach to this.

**Comment by jacobthill** [comment]
Ok I'm available to pair any time this afternoon. I'm not sure how to determine whether the issue is caused by traject or our transform code.

**Comment by jacobthill** [comment]
I'm inclined to ignore the code climate warning. This was an incredibly complex mapping challenge because the metadata was so inconsistent across collections. I don't see much value in adding a common config to put only the agg fields.

File: traject_configs/auc_first_date_for_range_config.rb [hunk 1]
```diff
@@ -1,5 +0,0 @@
-# frozen_string_literal: true
-
-# AUC collection p15795coll8 has undesired date info except in first dc:date field
-to_field 'cho_date_range_norm', extract_oai('dc:date'), first_only, strip, auc_date_range
-to_field 'cho_date_range_hijri', extract_oai('dc:date'), first_only, strip, auc_date_range, hijri_range
```
Comment (ndushay) [inline traject_configs/auc_first_date_for_range_config.rb]: it's fine that you don't want this as a standalone config, but it was put here to solve a problem with coll8 being not like the others ... did that concern go away?

**Comment by jacobthill** [comment]
Naomi, those lines were moved into the config for that collection including the comment

**Comment by aaron-collier** [comment]
Approving / Merging, though I think this warrants a good amount of refactoring to clean up the duplication. There does seem to be some difference in fields, but also a lot of those differences are literals, which we may want to move into settings to limit code duplication.

**Pull request merged**